src/Kros.AspNetCore/Extensions/HealthChecksBuilderExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Kros.Identity.Extensions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+
+namespace Kros.AspNetCore.Extensions
+{
+    public static class HealthChecksBuilderExtensions
```

Comment:
According to our code style, we require all public classes \ methods \ ... to contain a comment